src/core_plugins/kibana/public/management/sections/indices/_edit.js

```diff
@@ -67,7 +64,7 @@ uiModules.get('apps/management')
   });
 
   $scope.$watchCollection('indexPattern.fields', function () {
-    $scope.indexPattern.fields.filter(type => { type: 'conflict'; });
+    $scope.indexPattern.fields.filter(() => { type: 'conflict'; });
```

Comment:
thanks @kjbekkelund. that PR looks like it introduced a bug. I'll wait until this is merged, and then put up a new PR that fixes this issue in timelion. thx.